build/testdata/070.build.golden

```diff
@@ -4,8 +4,8 @@ foo(
         "aaa",
         "bbb",
     ] + CONSTANT + [
-        "ccc",
-        "ddd",
+        "//ccc",
+        "//ddd",
```

Comment:
why? This looks like a bug?

Reply:
You mean that `srcs` is interpreted as labels?

Reply:
oops, I missed the change in the .in file
lgtm